contrib/go-pg/pg.v10/example_test.go

```diff
@@ -0,0 +1,33 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package pg_test
+
+import (
+	"context"
+	"log"
+
+	pgtrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/go-pg/pg.v10"
+
+	"github.com/go-pg/pg/v10"
+)
+
+func Example() {
+	conn := pg.Connect(&pg.Options{
+		User:     "go-pg-test",
+		Database: "datadog",
+	})
+
+	// Wrap the connection with the APM hook
+	pgtrace.Wrap(conn)
+	// For correct tracing, must be query execute with context.
```

Comment:
Why? Won't it work without context? If not, let's correct the spelling a bit:
```suggestion
	// For queries to be traced, context must be used because ...
```